Mlem/App/Views/Root/Tabs/Settings/InteractionBarEditor/InteractionBarEditorView+Views.swift

```diff
@@ -47,6 +32,54 @@ extension InteractionBarEditorView {
         .paletteBorder(cornerRadius: Constants.main.mediumItemCornerRadius)
     }
     
+    @ViewBuilder
+    var postPreviewBody: some View {
+        HStack(alignment: .top, spacing: 8) {
+            RoundedRectangle(cornerRadius: Constants.main.smallItemCornerRadius)
+                .fill(palette.accent.opacity(0.6))
+                .frame(width: Constants.main.thumbnailSize, height: Constants.main.thumbnailSize)
+                .overlay {
+                    Image(systemName: "mountain.2.fill")
+                        .font(.system(size: 23))
+                        .foregroundStyle(.white)
+                }
+            
+            VStack(alignment: .leading, spacing: 5) {
+                MockTextView()
+                    .frame(maxWidth: .infinity)
+                    .frame(height: 15)
+                MockTextView()
+                    .frame(maxWidth: 200)
+                    .frame(height: 15)
+            }
+        }
+    }
+    
+    @ViewBuilder
+    var commentPreviewBody: some View {
+        HStack(spacing: 7) {
+            Image(systemName: Icons.personCircleFill)
+                .resizable()
+                .scaledToFit()
+                .symbolRenderingMode(.palette)
+                .foregroundStyle(palette.selectedInteractionBarItem, palette.neutralAccent.gradient)
+                .frame(width: Constants.main.smallAvatarSize, height: Constants.main.smallAvatarSize)
```

Comment:
Fair enough. What do you think about lower opacity?

```suggestion
                .frame(width: Constants.main.smallAvatarSize, height: Constants.main.smallAvatarSize)
                .compositingGroup()
                .opacity(0.5)
```

Reply:
Works for me 👍